app/templates/admin/content/system-images/subtopics.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="ui segment">
+  {{#each model as |subTopic|}}
+    <h4>{{subTopic.name}}</h4>
+    <img src="{{subTopic.placeholder.originalImageUrl}}" id="subTopicPlaceholder" alt="image">
```

Comment:
Why is an required in the first place? Remove it. (and even if it was required, an id should always be unique) 
And also, keep the alt value as '<subtopic name> placeholder'